index.js

```diff
@@ -132,85 +205,55 @@ async function localizeMessageShortcut({ shortcut, ack, context, payload }) {
+            convertedMessage.replace(/^|\n/g, '\n>')
+        });
+      } else {
+        //not in the channel, so we send as a modal
+        await app.client.views.open(
+          blocks.messageModal({
+            token: context.bot_token,
+            trigger_id: payload.trigger_id,
+            text: message,
+            origUser: shortcut.user.id,
+            helpText: `Hint: Want others to be able to see this? Invite <@U019XGT657V> to the channel.`
+          })
+        );
+      }
     } else {
-      await app.client.views.open({
-        // The token you used to initialize your app is stored in the `context` object
-        token: context.botToken,
-        trigger_id: payload.trigger_id,
-        view: {
-          type: 'modal',
-          title: {
-            type: 'plain_text',
-            text: 'Jonathan'
-          },
-          blocks: [
-            {
-              type: 'header',
-              text: {
-                type: 'plain_text',
-                text: "Here's that post in your timezone:",
-                emoji: true
-              }
-            },
-            {
-              type: 'section',
-              text: {
-                type: 'mrkdwn',
-                text: convertedMessage
-              }
-            },
-            {
-              type: 'divider'
-            },
-            {
-              type: 'context',
-              elements: [
-                {
-                  type: 'mrkdwn',
-                  text: `\n\nBy the way, you should ask <@${shortcut.message.user}> to trigger this on their own message: I'll reply in-thread and magically convert the times for everyone.`
-                }
-              ]
-            }
-          ]
-        }
-      });
+      // It's not the original user, so we show a modal
+      await app.client.views.open(
+        blocks.messageModal({
+          // The token you used to initialize your app is stored in the `context` object
+          token: context.bot_token,
+          trigger_id: payload.trigger_id,
+          text: message,
+          origUser: shortcut.user.id,
+          helpText: `\n\nBy the way, you should ask <@${shortcut.user.id}> to trigger this on their own message: I'll reply in-thread and magically convert the times for everyone.`
+        })
+      );
     }
     //send response message
   } catch (err) {
     console.error(err);
-
-    // Extract details of the error which are not stack or message
-    const { stack, code, message, ...error_details } = err;
-
-    Honeybadger.notify(
-      { stack, code, message },
-      {
-        context: {
-          timeMatches,
-          message: {
-            ts: shortcut.message.ts,
-            text: shortcut.message.text
-          },
-          channelId: shortcut.channel.id,
-          userId: shortcut.user.id,
-          teamId: shortcut.team.id,
-          error_details
-        }
-      }
-    );
```

Comment:
Because the bulk of this is just object creation, I think we should probably keep this in the function in-scope.